inc/commonitilobject.class.php

```diff
@@ -1834,6 +1834,21 @@ function post_addItem() {
          }
       }
 
+      // Handle deferred solution addition (for solution templates added by rule on ticket Add)
+      if (isset($this->input['_solutiontemplates_id'])) {
+         $template = new SolutionTemplate();
+         if ($template->getFromDB($this->input['_solutiontemplates_id'])) {
+            $solution = new ITILSolution();
+            $solution->add([
+               "itemtype" => static::getType(),
+               "solutiontypes_id" => $template->getField("solutiontypes_id"),
+               "content" => Toolbox::addslashes_deep($template->getField('content')),
```

Comment:
`CommonDBTM::getField()` should only be used for display purpose.

```suggestion
               "solutiontypes_id" => $template->fields["solutiontypes_id"],
               "content" => Toolbox::addslashes_deep($template->fields['content']),
```